tools/mage/js.go

```diff
@@ -215,7 +215,7 @@ func (js Js) Messages() error {
 	if err = os.MkdirAll(filepath.Join("pkg", "webui", "locales"), 0o755); err != nil {
 		return fmt.Errorf("failed to create locale directory: %w", err)
 	}
-	return execYarn(nil, os.Stderr, "babel", filepath.Join("pkg", "webui"))
+	return execYarn(nil, os.Stderr, "babel", filepath.Join("pkg", "webui", "console.js"), filepath.Join("pkg/webui/account.js"))
```

Comment:
It should be `"pkg", "webui", "account.js"`, like what you have for `console.js`.